src/postgraphile/cli.ts

```diff
@@ -422,6 +424,8 @@ if (noServer) {
     const rawMiddleware = postgraphile(pgConfig, schemas, postgraphileOptions)
     const middleware = pluginHook('cli:server:middleware', rawMiddleware, { options: postgraphileOptions })
     const server = createServer(middleware)
+    server.timeout = serverTimeout
```

Comment:
I'm not super comfortable using a property access to `timeout` when there's a setter method `setTimeout` present; however I'm struggling to tell from the docs if they do the same thing or not:

![screenshot 2018-05-03 17 33 43](https://user-images.githubusercontent.com/129910/39590208-23ad9662-4ef8-11e8-83c5-bbe6a71fc7a9.png)

Can you point to someone of authority stating that these are either equivalent, or that they're different and why you'd use one over the other?

Reply:
Hey @kronken, any idea regarding this? Are they different?

Reply:
Judging by the Node.js source they're equivalent:

https://github.com/nodejs/node/blob/79e63c5fc727254e3d1299c7d062b6caca6e9a86/lib/_http_server.js#L310-L315